.github/workflows/release.yml

```diff
@@ -1,32 +1,94 @@
-name: Release
+name: Release Gem
 
-on:
-  push:
-    tags:
-      - '*'
+on: workflow_dispatch
 
 jobs:
   release:
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'voxpupuli'
+    if: github.repository == 'puppetlabs/beaker-puppet'
```

Comment:
this is totally fine, but: For Vox Pupuli we switched from `github.repository` to `github.repository_owner` because it turned out people often copy and paste the action to new repos we have and forget to adjust it :D